app/jobs/email_privacy_policy_update_job.rb

```diff
@@ -0,0 +1,43 @@
+class EmailPrivacyPolicyUpdateJob < ApplicationJob
+  def perform(petition)
+    terminating = false
+
+    worker = trap("TERM") do
+      terminating = true
+      worker.call
+    end
+
+    Appsignal.without_instrumentation do
+      petition.signatures.find_each do |signature|
+        create_privacy_notification!(signature)
+        send_email(signature)
```

Comment:
It might be the case we need separate emails for creators/signers - will need to check with Ben.